### DISCUSSION TITLE
Jenkins Pipeline 介绍

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#376] [Announcements]
# Jenkins Pipeline 介绍

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2019/08/30/jenkins-pipeline-intro/

**Comment by Xuanwo** [comment]
令人感慨，19 年的我还是个 Jenkins Scripts Boy，甚至在准备自制 DSL，做分享；如今已经是 Jenkins 头号黑粉了。。。

**Reply by lv123123long** [reply]
发生了什么，，我现在也是jenkins boy
